deepblink/networks/inception.py

```diff
@@ -0,0 +1,125 @@
+# pylint: skip-file
```

Comment:
Looks complicated but is fine. If FCN worked better than this (also keep n_params in mind) then there's no need to invest too much time here.